src/data/api/ApiRequest.ts

```diff
@@ -25,7 +25,7 @@ async function getRequest(
     })
     .catch((error) => {
       console.log('Error with fetching data: ' + error);
-      return null;
```

Comment:
Waarom niet gewoon houden op null?

Reply:
Omdat ik die status nodig heb.

Reply:
Is het dan niet handiger om een object aan te maken met status code, evt als enum of number en ook de message die de API terug geeft. Dit ziet er namelijk hacky uit

Reply:
Sure. Zal ik nog even doen.